pkg/benchmarks/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
this is a severe security error and must be undone

Reply:
Since the core package has no dependencies, this just adds constant churn to track the dev tools.  We can talk about the cli.